CareKit/CareKit/iOS/Contact/View Controllers/OCKContactViewResponder.swift

```diff
@@ -159,6 +159,16 @@ final class OCKContactViewResponder: NSObject,
             action: #selector(dismissViewController)
         )
         
+        /*
+         TODO: Remove in the future. Explicitly setting the tint color here to support
+         current developers that have a SwiftUI lifecycle app and wrap this view
+         controller in a `UIViewControllerRepresentable` implementation...Tint color
+         is not propagated...etc.
+         */
+        let tintColor = contactViewController.determineTintColor(from: contactView)
```

Comment:
All of the tintColor changes are simply the updates I added in #676, but in the correct places for 3.0.0